ifrs17/Utils/ImportCalculationMethods.ipynb

```diff
@@ -246,6 +247,31 @@
             "execution_count": 0,
             "outputs": []
         },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## Defer Acquisition Cost"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "public static double[] Extrapolate(this double[] cashflowValues, string amountType, int contractTerm, double[] dacAmortizationFactors)",
```

Comment:
I would not do this now... 
I consider this part of the DAC calculation, let's do it (if necessary) in that PR.

Reply:
removed